OpenRA.Mods.Common/Traits/Render/WithProductionOverlay.cs

```diff
@@ -31,6 +31,8 @@ public class WithProductionOverlayInfo : ITraitInfo, Requires<RenderSpritesInfo>
 		[Desc("Custom palette is a player palette BaseName")]
 		public readonly bool IsPlayerPalette = false;
 
+		public readonly bool PauseOnLowPower = false;
```

Comment:
If neither the `UpgradeWhenDisabled` nor the `LowPowerUpgrade` already exist than this isn't deprecated yet. Make a ticket for your future plans instead.